### PR TITLE
initial version of spectraperf

### PR DESCRIPTION
**Pull request by corpaul**

### CODE REVIEW

File: spectraperf/DatabaseHelper.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+'''
```
Comment (whirm) [inline spectraperf/DatabaseHelper.py]: Lets use lowercase for all the files

File: scripts/run_in_env.sh [hunk 2]
```diff
@@ -44,6 +44,9 @@ echo "Project root is: $PROJECTROOT"
 # Load env variables for this experiment
 source experiment_vars.sh
 
+# Update PYTHONPATH
+export PYTHONPATH=$PYTHONPATH:$PROJECTROOT/spectraperf
```
Comment (whirm) [inline scripts/run_in_env.sh]: Better to add $PROJECTROOT to the python path and update the module imports. That way we avoid name conflicts between projects.

File: tests/testPerformanceProfile.py [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+        #p.addToRange("test2", 10)
+        #p.addToRange("test2", 20)
+
+
+        fits = p.fitsProfile(sess1)
+        self.assertEqual(fits["test1"], 1)
+
+        sess1.addStacktrace(s2)
+        # should raise AssertionError because we did not add a range for test2 yet
+       # with self.assertRaises(AssertionError):
```
Comment (whirm) [inline tests/testPerformanceProfile.py]: bad indentation levels

File: scripts/run_in_env.sh (hunk 2 above)
Comment (corpaul) [inline scripts/run_in_env.sh]: where are those and what should I change?

**Pull request merged**